var/spack/repos/builtin/packages/benchmark/package.py

```diff
@@ -30,7 +31,8 @@ class Benchmark(CMakePackage):
                     'MinSizeRel', 'Coverage'))
 
     depends_on("cmake@2.8.11:", type="build", when="@:1.1.0")
-    depends_on("cmake@2.8.12:", type="build", when="@1.2.0:")
+    depends_on("cmake@2.8.12:", type="build", when="@1.2.0:1.4")
```

Comment:
The range should be @1.2.0:1.4.99 otherwise 1.4.1 will not match.

Reply:
Actually, "1.4" will match all versions of "1.4.x". "1.4.0" would limit it.

Reply:
This was new to me. I had made the same assumption.

Reply:
OK. As long and the older version can be built I am fine with this change.